legacy/src/main/java/org/opensearch/sql/legacy/plugin/RestSQLQueryAction.java

```diff
@@ -61,6 +65,14 @@ public class RestSQLQueryAction extends BaseRestHandler {
    */
   private final Settings pluginSettings;
 
+  /**
+   * Captured error message to aggregate diagnostics
+   * for both legacy and new SQL engines.
+   * This member variable and it's usage can be deleted once the
+   * legacy SQL engine is deprecated.
```

Comment:
Is there an issue in GitHub for this?  I wonder if there's something we could track.

Reply:
I was unable to find a related github issue. @Yury-Fridlyand Are you aware of any related issue?

Reply:
No, I also failed to find an issue.